pkgs/by-name/me/meson/package.nix

```diff
@@ -11,15 +11,36 @@
 , ninja
 , pkg-config
 , python3
+, python3Packages
```

Comment:
The [eval failure](https://gist.github.com/GrahamcOfBorg/0ca2693b1cf51e60af10c3980a7b0b67) is because darwin bootstrap uses meson and doesn't override `python3Packages` ([but does python3](https://github.com/NixOS/nixpkgs/blob/3ed4da05f610b2af67fa50c56e214c7909b11893/pkgs/stdenv/darwin/default.nix#L625))

Either you use `python3.pkgs` or we do :arrow_down: 
```diff
diff --git a/pkgs/stdenv/darwin/default.nix b/pkgs/stdenv/darwin/default.nix
index 588e3b03d92b..1c6e7d9eda0c 100644
--- a/pkgs/stdenv/darwin/default.nix
+++ b/pkgs/stdenv/darwin/default.nix
@@ -907,6 +907,7 @@ assert bootstrapTools.passthru.isFromBootstrapFiles or false; # sanity check
 
         # The bootstrap Python needs its own `pythonAttr` to make sure the override works properly.
         python3 = self.python3-bootstrap;
+        python3Packages = self.python3-bootstrap.pkgs;
         python3-bootstrap = super.python3.override {
           self = self.python3-bootstrap;
           pythonAttr = "python3-bootstrap";
```